getting-started/ci-setup/github-actions/custom-ci-build-id-for-reruns.md

```
@@ -0,0 +1,40 @@
+    # if a custom CI build id is used, set "previous-ci-build-id" accordingly
+    previous-ci-build-id: currents-${{ github.run_id }}-<%= ${{ github.run_attempt }} - 1 %>
+    pw-output-dir: basic/test-results
+    matrix-index: ${{ matrix.shard }}
+    matrix-total: ${{ strategy.job-total }}
+```
+{% endcode %}
+
+For orchestrated runs, keep the `or8n` input and skip the matrix values:
+
+{% code overflow="wrap" %}
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)